components/browser/tabstray/src/main/java/mozilla/components/browser/tabstray/TabsAdapter.kt

```diff
@@ -17,7 +17,8 @@ import mozilla.components.support.base.observer.ObserverRegistry
  */
 @Suppress("TooManyFunctions")
 class TabsAdapter(
-    delegate: Observable<TabsTray.Observer> = ObserverRegistry()
+    delegate: Observable<TabsTray.Observer> = ObserverRegistry(),
+    private val layoutId: Int = R.layout.mozac_browser_tabstray_item
```

Comment:
Add `@LayoutRes` to the param [like so](https://github.com/mozilla-mobile/android-components/blob/master/components/browser/awesomebar/src/main/java/mozilla/components/browser/awesomebar/layout/SuggestionLayout.kt#L29).

This gives the linter hints for type mismatches.